packages/web-forms/src/components/widgets/IconSVG.vue

```diff
@@ -0,0 +1,125 @@
+<script setup lang="ts">
+import { computed } from 'vue';
+import {
+	mdiAlert,
```

Comment:
Is the `mdi` prefix needed here? Is it because additional icon sources could be specified?

Reply:
Yes, it's needed; that's how the library exports them:
<img width="300" alt="Screenshot 2025-04-28 at 10 15 18 AM" src="https://github.com/user-attachments/assets/5faacfb2-d8ca-405a-a9bb-2259fe389bca" />

Otherwise, it can't find the icon:
<img width="300" alt="Screenshot 2025-04-28 at 10 15 07 AM" src="https://github.com/user-attachments/assets/92726d7d-71cf-45cd-ae83-cbb0d731eda3" />